### PR TITLE
Bugfix/batch pb6 fixes 08 2024

### PR DESCRIPTION
**Pull request by lopezt-unity**
**DO NOT FORGET TO INCLUDE A CHANGELOG ENTRY**

### Fixed

- [PBLD-151] Fixed a bug where the Action Settings overlay would not be displayed when used the first time.
- [PBLD-141] Fixed a bug where selection would be incorrect when entering PrefabStage.
- [PBLD-142] Fixed a bug where it was possible to trigger the probuilderize action twice on the same object.
- [PBLD-150] Fixed a bug where the orientation handles would not be useful when manipulating a cube shape.
- [PBLD-149] Fixed a bug where the ProBuilder edit modes toggle could be disabled.

### Added

- [PBLD-152] Added some relevant Unity selection actions in the ProBuilder contextual menu.

### Links

**Jira:** 
https://jira.unity3d.com/browse/PBLD-141
https://jira.unity3d.com/browse/PBLD-142
https://jira.unity3d.com/browse/PBLD-149
https://jira.unity3d.com/browse/PBLD-150
https://jira.unity3d.com/browse/PBLD-151
https://jira.unity3d.com/browse/PBLD-152

### Comments to Reviewers

[List known issues, planned work, provide any extra context for your code.]

### CODE REVIEW

**Review by modrimkus-unity — APPROVED** [review]
Left minor note. LGTM!

File: Content/Resources/Materials/ProBuilderDefault.mat [hunk 1]
```diff
@@ -59,7 +59,7 @@ Material:
   m_LightmapFlags: 1
   m_EnableInstancingVariants: 0
   m_DoubleSidedGI: 0
-  m_CustomRenderQueue: 2000
+  m_CustomRenderQueue: -1
   stringTagMap:
```
[the remark this answers is not in the document]
Comment (lopezt-unity) [reply Content/Resources/Materials/ProBuilderDefault.mat:63]: Damn PB mat.... :P

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Fixed
+
+- [PBLD-151] Fixed a bug where the Action Settings overlay would not be displayed when used the first time.
+- [PBLD-141] Fixed a bug where selection would be incorrect when entering PrefabStage.
```
Comment (claudiaantoun) [inline CHANGELOG.md:13]:
```suggestion
- [PBLD-141] Fixed a bug where the selection would be incorrect when entering PrefabStage.
```

**Review by claudiaantoun — APPROVED** [review]
LGTM! Just a tiny nitpick on the changelog comment.

**Pull request merged**